ui/packages/atlasmap/src/Atlasmap/AtlasmapProvider.tsx

```diff
@@ -278,15 +277,15 @@ export const AtlasmapProvider: FunctionComponent<IAtlasmapProviderProps> = ({
   );
 
   const onSubUpdate = useCallback(
-    function onSubUpdateCb(caller: string) {
-      console.log(
-        "onUpdates",
-        caller,
-        "initialized",
-        initializationService.cfg.initCfg.initialized,
-        "errors",
-        initializationService.cfg.initCfg.initializationErrorOccurred,
-      );
+    function onSubUpdateCb(_caller: string) {
+      // console.log(
```

Comment:
I'd rather keep this log, it's helpful to grasp what's going on with the backend sometimes.

Reply:
Yes, I'll uncomment that. I just did that because it's annoying when I'm debugging other stuff using the console.